tests/openwisp2/sample_check/models.py

```diff
@@ -0,0 +1,8 @@
+from openwisp_monitoring.check.base.models import AbstractCheck
+from swapper import swappable_setting
+
+
+class Check(AbstractCheck):
```

Comment:
Take a look at this: https://github.com/openwisp/openwisp-firmware-upgrader/blob/master/tests/openwisp2/sample_firmware_upgrader/models.py#L12-L16

It's a good way to test the extensibility.
We just have to ensure the field is shown in the admin with an extra test for each admin class.

Apply this to all models please.